### ISSUE TITLE
TypeError: Cannot create property 'value' on boolean 'false'

### ISSUE THREAD

**Issue by phet-steele** [phetsims/ohms-law#128]
There are billions of ways to get this bug, after which the sim will freeze. There are two general ways:

1. Use keyboard nav to open the dialogue of your choice (phet menu/keyboard help are the only two in this sim, technically).
2. Sim will freeze with the dialogue stuck open.

or

1. Use your mouse to open the dialogue of your choice (keyboard help/about/check for updates but not the phet menu b/c that gets closed for some reason [bug?]).
2. Tab to highlight the dialogue's close button.
3. Space/Enter to click the button.
4. Sim will freeze with the dialogue successfully closed.

Each way gives:

```
Uncaught TypeError: Cannot create property 'value' on boolean 'false'
    at ohms-law_all_phet_debug.html:106143
    at callback (ohms-law_all_phet_debug.html:101724)
    at Timer.emit (ohms-law_all_phet_debug.html:44083)
    at Timer.emit1 (ohms-law_all_phet_debug.html:44100)
    at first (ohms-law_all_phet_debug.html:135171)
    at Emitter.emit (ohms-law_all_phet_debug.html:44083)
    at Sim.stepSimulation (ohms-law_all_phet_debug.html:135878)
    at Sim.stepOneFrame (ohms-law_all_phet_debug.html:135859)
    at Sim.runAnimationLoop (ohms-law_all_phet_debug.html:135842)
```

Seen on Win 10. Chrome. For phetsims/QA/issues/289.

<details>
<summary>Troubleshooting Information</summary>

URL: https://phet-dev.colorado.edu/html/ohms-law/1.4.0-dev.26/phet/ohms-law_en_phet.html
Version: 1.4.0-dev.26 2019-02-16 00:29:23 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36
Language: en-US
Window: 1920x938
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4095
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
</details>

**Comment by jessegreenberg** [comment]
Hmm, this isn't happening for me in master.

**Comment by jessegreenberg** [comment]
Verified this is happening in the dev version, but not in master.

**Comment by jessegreenberg** [comment]
I found a bad line in the dev version that caused this that was present in my working copy at the time of the build
![image](https://user-images.githubusercontent.com/6396244/52979962-fab07700-33a5-11e9-94b8-450935ab329f.png)

 It isn't something that is committed so I am surprised I was able to publish a dev version with this.

Should be `this.a11yClickingProperty.value = false;`

**Comment by jessegreenberg** [comment]
In master, I notice that focus isn't passed around correctly anymore because of this new block for "handling a11y interrupt". Ill make a new issue to investigate, but this error specifically can be closed.
